%-------------------------
% Based on Sourabh Bajaj's template (https://github.com/sb2nov/resume). Credit to him.
%-------------------------

\documentclass[letterpaper,11pt]{article}

\usepackage{latexsym}
\usepackage[empty]{fullpage}
\usepackage{titlesec}
\usepackage{marvosym}
\usepackage[usenames,dvipsnames]{color}
\usepackage{verbatim}
\usepackage{enumitem}
\usepackage[hidelinks]{hyperref}
\usepackage{fancyhdr}
\usepackage[english]{babel}
\usepackage{geometry}
\geometry{
  a4paper,
  left=0.7in,
  right=0.7in,
  top=0.7in,
  bottom=0.7in
}

\usepackage{tgpagella} % text only
%\usepackage[T1]{fontenc}
% \usepackage{fontspec}
% \setmainfont{Georgia}
% \usepackage{avant}
% \renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
% \usepackage[T1]{fontenc}


\pagestyle{fancy}
\fancyhf{} % clear all header and footer fields
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}

% % Adjust margins
% \addtolength{\oddsidemargin}{-0in}
% \addtolength{\evensidemargin}{0in}
% \addtolength{\marginparsep}{0in}
% \addtolength{\textwidth}{1in}
% \addtolength{\topmargin}{-0.3in}
% \addtolength{\textheight}{1.0in}

\urlstyle{same}

\raggedbottom
\raggedright
\setlength{\tabcolsep}{0in}

% Sections formatting
\titleformat{\section}{
  \vspace{-4pt}\scshape\raggedright\Large
}{}{0em}{}[\color{black}\titlerule \vspace{-5pt}]

%-------------------------
% Custom commands
\newcommand{\resumeItem}[2]{
  \item {#2\vspace{-2pt}} %\textbf{#1}{ #2 \vspace{-2pt}}
}

% \newcommand{\resumeSubheading}[4]{
%   \vspace{-1pt}\item
%     \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
%       \textbf{#1} & #2 \\
%       \textit{\small#3} & \textit{\small #4} \\
%     \end{tabular*}\vspace{-5pt}
% }

\newcommand{\resumeSubheading}[4]{
  \vspace{-1pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1}, {#2} (#3) & \textit{#4} \\
    \end{tabular*}\vspace{-5pt}
}

\newcommand{\resumeSubheadingOneLine}[2]{
  \vspace{-1pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
    \end{tabular*}\vspace{-5pt}
}

\newcommand{\resumeSubItem}[2]{\item \textbf{#1}{ #2\vspace{-2pt}}\vspace{-4pt}}

\renewcommand{\labelitemii}{$\circ$}

\newcommand{\resumeSubHeadingListStart}{\begin{itemize}[leftmargin=*]}
\newcommand{\resumeSubHeadingListEnd}{\end{itemize}}
\newcommand{\resumeItemListStart}{\begin{itemize}[leftmargin=*]} % ,noitemsep,topsep=0pt
\newcommand{\resumeItemListEnd}{\end{itemize}\vspace{-5pt}}

%-------------------------------------------
%%%%%%  CV STARTS HERE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%


\begin{document}

%----------HEADING-----------------
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
  \textbf{\href{https://zeqianli.me}{\Large Zeqian Li}} &  \href{https://zeqianli.me}{https://zeqianli.me} $\mid$ zeqianli.chicago@gmail.com $\mid$ Chicago, IL
\vspace{-10pt}
\end{tabular*}

%-----------EDUCATION-----------------
\section{Education}
  \resumeSubHeadingListStart
    \resumeSubheading
    {Graduate Student}{University of Chicago}{Chicago, IL}{Expected Aug 2023}
    \vspace{-12pt}
    \resumeSubheading
    {Ph.D in Physics}{University of Illinois at Urbana-Champaign}{Champaign, IL}{Expected Aug 2023}
    \resumeSubHeadingListStart
    \item Center for Physics of Living Cells Fellow (\textit{2018 - 2020})
\resumeSubHeadingListEnd

      \vspace{-8pt}
    \resumeSubheading
    {B.S in Physics}{Hong Kong Baptist University}{Hong Kong}{Sep 2014 -- July 2018}
    \resumeSubHeadingListStart
    \item Hong Kong Special Administrative Region Government Scholarship (\textit{2015 - 2018})
    \item Scholastic Award (\textit{2018}) 
    % \end{tabular*}\vspace{-5pt}
\resumeSubHeadingListEnd
    
    \vspace{-10pt}
  \resumeSubHeadingListEnd
  
%-----------EXPERIENCE-----------------
%-----------Publications----------------

\section{Experience}
  \resumeSubHeadingListStart

  \resumeSubheading
  {Seppe Kuehn lab (University of Chicago / UIUC)}{Research Assistant}
  {IL}{July 2019 - Aug 2023}
    \resumeItemListStart
    \resumeItem{Research:}{Led and collaborated in 4 research projects to study the fundamental principles behind microbial communities, combining experimental design, data-driven research and theoretical modeling. Will result in four publications and multiple conference presentations.}
    \resumeItem{Experimental design:}{Designed experiments and conducted whole-genome sequencing on soil isolated microbes.}
    \resumeItem{Statistical modeling:}{Used machine learning to achieve state-of-the-art prediction of microbial carbon utilization.}
    \resumeItem{Bioinformatics:}{Built bioinformatic pipelines for large multi-omics datasets in high-speed computing clusters.}
    \resumeItem{Data collection:}{Scrapped and cleaned datasets from online databases to statistically model microbial metabolism.}
    \resumeItem{Theoretical modeling:}{Created mathematical models of biological buffers and microbial respiration/photosynthesis.}
    \resumeItem{Hardware:}{Constructed and troubleshooted microcontroller-based experimental devices. }
    
\resumeItemListEnd
  %\resumeItemListStart

      % \resumeItem{Machine learning predicts microbial metabolic traits from genomes:}
      % {We studied the essential evolutionary determinants of microbial carbon metabolism. We showed that phylogeny strongly predicted microbial carbon utilization and large datasets would enable machine learning models to make mechanistic predictions. 
      % }

      % \resumeItem{Multi-omics patterns in the Yellowstone hot spring microbial mats:} 
      % {Using metagenome, metatranscriptome and single-cell amplified genome data, we showed that genome organization in the Yellowstone microbial mats is constrained by co-expression and is connected to extensive recombination.}

      % \resumeItem{Other Kuehn Lab projects:}
      % {An innovative way to quantify microbial respiration and photosynthesis (de Jesus Astacio et al, PNAS 2021), predicting media buffering capacity (Gopalakrishnappa et al, in preparation), and evolutionary structures of the denitrification pathway (Crocker et al, in preparation).
      % }
      % \vspace{-5pt}

    %\resumeItemListEnd

    \resumeSubheading
      {Upward Farms}{Microbial Research Associate}{Brooklyn, NY}{May 2022 - Aug 2022}
        \resumeItemListStart
      % \resumeItem{Improve hydroponic crop yields through microbial transplanting:}{I led a research project on microbial association with hydroponic plants. Using 16S sequencing, we showed that microbial composition strongly correlated with plant growth. We identified potential growth-promoting microbes through statistical modeling.}
      \resumeItem{Research:}{Led an innovative research to improve hydroponic crop yields by manipulating plant microbiome.}
      %\item Led a innovative research project to improve hydroponic crop yields by manipulating plant microbiome.
      \resumeItem{Bioinformatics:}{Used statistical modeling and 16S sequencing to identified plant growth-promoting bacteria.}
      \resumeItem{Production-level software:}{Built and unit-tested two software prototypes in AWS: a Snakemake pipeline to streamline NGS sequencing data analysis and a web-based R\&D experiment management portal.} % Both will be incorporated into production.}
      % \item Identified potential growth-promoting microbes through 16S data analysis and statistical modeling.
      % \item Built and unit-tested two software prototypes: a Snakemake pipeline to streamline NGS whole-genome sequencing data analysis and a web-based R\&D experiment management portal. Both will be incorporated into production.
        % \resumeItem{Other wet lab experiments:}{Nanopore sequencing, crop phenotyping, and sample collection.}
        \resumeItem{Experiments:}{Conducted experiments in nanopore sequencing, crop phenotyping, and sample collection.}
        \resumeItem{Team experience:}{Collaborated closely with the R\&D team using Git and project management tools.}
    \resumeItemListEnd
    
    % \resumeSubheading
    %   {The Abdus Salam International Centre for Theoretical Physics}{Trieste, Italy}
    %   {Spring College on the Physics of Complex Systems}{Mar 2018}

    \vspace{-1pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{The Abdus Salam International Centre for Theoretical Physics} &  {\textit{Mar 2018}} \\ Spring College on the Physics of Complex Systems (Trieste, Italy) & 
    \end{tabular*}\vspace{-5pt}
    \resumeItemListStart
    \resumeItem{Coursework:}{Completed graduate-level courses in reinforcement learning, statistical physics, and biophysics.}
    \resumeItemListEnd

    \resumeSubheading
    {Hong Kong Baptist University}{Research Assistant}
    {Hong Kong}{July 2015 - June 2018}
      \resumeItemListStart
      \resumeItem{Computational modeling:}{Built innovative machine learning models inspired by neuroscience and statistical physics.}
      \resumeItem{Data-driven research:}{Conducted multiple data-driven research projects across various biological systems.}
      \resumeItem{Teaching:}{Taught discussion sessions of two college introductory physics courses.}
      
  \resumeItemListEnd
    
    % \vspace{-1pt}\item
    % \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}}
    %   {\textbf{Teaching assistant}: Introductory physics courses in college.}  \\
    % \end{tabular*}\vspace{-5pt}
    
    
    %\\ Took five graduate courses with grade E (excellent).}
    %   \resumeItemListStart
    %     \resumeItem{Computational capacities of spiking neural networks with critical avalanches}{}
    %       % {\\
    %       % We developed a spiking neural network model to perform computational tasks under supervision. The model, inspired by Liquid State Machine and excitation-inhibition balanced neurons, showed critical behaviors. We studied roles of criticality in neural computation.}
    %     \resumeItem{Cell adjacency relationships in C. elegans cell migration}{}
    %       %{\\
    %       %We studied C. elegans’ early embryonic development by investigating cell adjacency relationships. We showed that cell contacts were deterministic across wild-type individuals.}
    %     \resumeItem{Feedback connections on C. elegans neural signal flow}{}
    %   \resumeItemListEnd
        
  \resumeSubHeadingListEnd



% %-----------TEACHING-----------------
% \section{Teaching Experience}
% \resumeSubHeadingListStart

%   \resumeSubheadingOneLine{Hong Kong Baptist University}{Teaching assistant for introductory physics courses.}
  
% \resumeSubHeadingListEnd


%\section{Publications}

% \resumeSubHeadingListStart
%     \item \textbf{Zeqian Li}, Ahmed Selim, Seppe Kuehn. ``Predict microbial metabolic traits from genomes.'' \\ \textit{In preparation} (2023).
%     \item Chandana Gopalakrishnappa, \textbf{Zeqian Li}, Seppe Kuehn. ``Environmental modulators of algae-bacteria interactions at scale.'' \textit{In preparation} (2023).
%     \item Luis Miguel de Jesús Astacio$^*$, Kaumudi H. Prabhakara$^*$, \textbf{Zeqian Li}, Harry Mickalide, Seppe Kuehn. ``Closed microbial communities self-organize to persistently cycle carbon.''  \textit{Proceedings of the National Academy of Sciences} 118, no. 45 (2021): e2013564118. 
% \resumeSubHeadingListEnd


% \section{Awards and honors}
% \resumeSubHeadingListStart
% \vspace{-1pt}\item
%     \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
%       \textbf{Center for Physics of Living Cells (CPLC) Fellow} & UIUC, 2018-2020 \\
%     \end{tabular*}\vspace{-5pt}
% \vspace{-1pt}\item
%     \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
%       \textbf{HKSAR Government Scholarship} & Hong Kong, 2015-2018 \\
%     \end{tabular*}\vspace{-5pt}
% \vspace{-1pt}\item
% \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
%   \textbf{Scholastic Award} & Hong Kong Baptist University, 2018 \\
% \end{tabular*}\vspace{-5pt}

%\resumeSubHeadingListEnd


%--------PROGRAMMING SKILLS------------
\section{Skills}
 \resumeSubHeadingListStart
   \resumeSubItem{Data science:}{Machine learning, applied statistics, data collection and cleaning, web scrapping, data visualization, remote computing, reinforcement learning, deep learning.}
   \resumeSubItem{Bioinformatics:}{Snakemake, 16S, metagenomics, transcriptomics, single-cell amplified genome, long-read sequencing, databases (KEGG, NCBI, UniProt, Pfam, BioCyc).}
   \resumeSubItem{Experimental microbiology:}{Experimental design, web lab, next-generation sequencing, nanopore sequencing, DNA extraction, common assays, microcontrollers (Arduino, Raspberry Pi), electronics.}
   \resumeSubItem{Computational biology and physics:}{Computational neuroscience, signal analysis, image analysis, dynamical systems,  numerical simulation, statistical physics.}
   \resumeSubItem{Software:}{Python (Numpy, Pandas, Scikit-Learn, Seaborn, Matplotlib, Dash/Plotly, Jupyter), MongoDB, R, Git, unit-testing, Linux, Bash, Java, JavaScript, \LaTeX, project management.}
 \resumeSubHeadingListEnd


%-------------------------------------------
\end{document}
