%-------------------------
% Based on Sourabh Bajaj's template (https://github.com/sb2nov/resume). Credit to him.
%-------------------------

\documentclass[letterpaper,11pt]{article}

\usepackage{latexsym}
\usepackage[empty]{fullpage}
\usepackage{titlesec}
\usepackage{marvosym}
\usepackage[usenames,dvipsnames]{color}
\usepackage{verbatim}
\usepackage{enumitem}
\usepackage[hidelinks]{hyperref}
\usepackage{fancyhdr}
\usepackage[english]{babel}
\usepackage[papersize={8.5in,11in}]{geometry}
\geometry{
  left=0.7in,
  right=0.7in,
  top=0.7in,
  bottom=0.7in
}

\usepackage{tgpagella} % text only
%\usepackage[T1]{fontenc}
% \usepackage{fontspec}
% \setmainfont{Georgia}
% \usepackage{avant}
% \renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
% \usepackage[T1]{fontenc}


\pagestyle{fancy}
\fancyhf{} % clear all header and footer fields
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}

% % Adjust margins
% \addtolength{\oddsidemargin}{-0in}
% \addtolength{\evensidemargin}{0in}
% \addtolength{\marginparsep}{0in}
% \addtolength{\textwidth}{1in}
% \addtolength{\topmargin}{-0.3in}
% \addtolength{\textheight}{1.0in}

\urlstyle{same}

\raggedbottom
\raggedright
\setlength{\tabcolsep}{0in}

% Sections formatting
\titleformat{\section}{
  \vspace{-4pt}\scshape\raggedright\Large
}{}{0em}{}[\color{black}\titlerule \vspace{-5pt}]

%-------------------------
% Custom commands
\newcommand{\resumeItem}[2]{
  \item {#2\vspace{-2pt}} %\textbf{#1}{ #2 \vspace{-2pt}}
}

% \newcommand{\resumeSubheading}[4]{
%   \vspace{-1pt}\item
%     \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
%       \textbf{#1} & #2 \\
%       \textit{\small#3} & \textit{\small #4} \\
%     \end{tabular*}\vspace{-5pt}
% }

\newcommand{\resumeSubheading}[4]{
  \vspace{-1pt}\item[]
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1}, {#2} (#3) & \textit{#4} \\
    \end{tabular*}\vspace{-5pt}
}

\newcommand{\resumeSubheadingOneLine}[2]{
  \vspace{-1pt}\item[]
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
    \end{tabular*}\vspace{-5pt}
}

\newcommand{\resumeSubItem}[2]{\item[] \textbf{#1}{ #2\vspace{-2pt}}\vspace{-4pt}}

\renewcommand{\labelitemii}{$\circ$}

\newcommand{\resumeSubHeadingListStart}{\begin{itemize}[leftmargin=0in]}
\newcommand{\resumeSubHeadingListEnd}{\end{itemize}}
\newcommand{\resumeItemListStart}{\begin{itemize}[leftmargin=*]} % ,noitemsep,topsep=0pt
\newcommand{\resumeItemListEnd}{\end{itemize}\vspace{-5pt}}

%-------------------------------------------
%%%%%%  CV STARTS HERE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%





\begin{document}

%----------HEADING-----------------
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
  \textbf{\href{https://zeqianli.me}{\Large Zeqian Li}} &  zeqianli.bio@gmail.com $\mid$ 217-377-7442
\vspace{-10pt}
\end{tabular*} 
% &  \href{https://zeqianli.me}{https://zeqianli.me}
%-----------EDUCATION-----------------
\section{Education}
  \resumeSubHeadingListStart
    % \resumeSubheading
    % {Graduate Research Assistant}{University of Chicago}{Chicago, IL}{Expected July 2023}
    % \vspace{3pt}
    % \vspace{-12pt}
    % \resumeSubheading
    % {Graduate Research Assistant}{University of Chicago}{Chicago, IL}{Expected July 2023}
    % \vspace{3pt}
    % % \vspace{-12pt}
    \resumeSubheading
    {Ph.D in Physics}{University of Illinois at Urbana-Champaign}{Champaign, IL}{Aug 2018 - Aug 2023}
      \resumeItemListStart
      \item Center for Physics of Living Cells Fellow (\textit{2018 - 2020})
      \resumeItemListEnd   
    \vspace{-2pt}
    \resumeSubheading
    {B.S in Green Energy Science}{Hong Kong Baptist University}{Hong Kong}{Sep 2014 -- July 2018}
      \resumeItemListStart
      \item Hong Kong Special Administrative Region Government Scholarship (\textit{2015 - 2018})
      \item Scholastic Award (\textit{2018}) 
      % \end{tabular*}\vspace{-5pt}
      \resumeItemListEnd    %\vspace{-10pt}
      \vspace{-2pt}
  \resumeSubHeadingListEnd
  \vspace{-10pt}
%-----------EXPERIENCE-----------------
%-----------Publications----------------

\section{Research and Professional Experience}
  \resumeSubHeadingListStart

  \resumeSubheading
  {BillionToOne}{Senior Bioinformatics Scientist, Oncology}
  {CA}{Aug 2023 - Present}
    \resumeItemListStart
    \resumeItem{}{Joined as the first bioinformatic scientist in BillionToOne oncology and built the bioinformatics roadmap for BillionToOne liquid biopsy assays, Northstar Select and Northstar Response}
    \resumeItem{}{Coordinated a 30+ people cross-functional CLIA laboratory to process 2000+ clinical samples. Regularly conducted custom NGS data analysis, performed quality control, and troubleshooted wet-lab processes}
    \resumeItem{}{Launched Northstar Response V1.1, a bioinformatic enhancement that boosted performance by 5x}
    \resumeItem{}{Launched Northstar Select Nextflow pipeline (3x faster) and reduced turnaround-time by 1 day}
    \resumeItem{}{Launched a full-stack oncology data dashboard web-app (NiceGUI, AWS)}
    \resumeItem{}{Led continuous algorithm development for the Northstar products (MSI caller, CNV caller, QC metrics)}
    \resumeItem{}{Led continous software development to scale-up sample processing capacity by $>$100X}
    \resumeItem{}{Grew the bioinformatics team by 6X. Facilitated hiring of 15+ full-time roles in BillionToOne.}
    \resumeItem{}{Worked with executive team in decision-making, facilitated commercial growth, and delivered accurate results to 2000+ cancer patients}
    \resumeItemListEnd
  \resumeSubheading
  {Seppe Kuehn lab (University of Chicago / UIUC)}{Research Assistant}
  {IL}{July 2019 - July 2023}
    \resumeItemListStart
    \resumeItem{Experimental design:}{Designed and conducted end-to-end genotyping (shotgun whole-genome sequencing) and phenotyping (carbon utilization assay) experiments on $100+$ novel environmental microbes}
    \resumeItem{Statistical modeling:}{Used machine learning to achieve state-of-the-art prediction of microbial carbon utilization, combining experimental data and large-scale web-scrapped datasets with over $4000$ bacterial genomes}
    \resumeItem{Bioinformatics:}{Built custom bioinformatic pipelines (Snakemake) on high-performance computing clusters to analyze over $10$TBs of multi-omics NGS data spanning more than $1000$ samples}
    \resumeItem{Theoretical modeling:}{Created accurate mathematical models for two systems (microbial respiration/photosynthesis and buffering capacity of complex biological media) and validated the models in experiments}
    \resumeItem{Hardware:}{Constructed microcontroller-based (Raspberry Pi) experimental devices and troubleshot Python-based software to interface sensors, PID controllers, and other electronic components}
    
    \resumeItemListEnd

    \resumeSubheading
      {Upward Farms}{Microbial Research Associate}{Brooklyn, NY}{May 2022 - Aug 2022}
        \resumeItemListStart
      \resumeItem{Research:}{Led end-to-end experiment to improve hydroponic crop yields by plant microbiome maniputation. Used statistical modeling and 16S sequencing to identify potential plant growth-promoting bacteria}
      \resumeItem{Production-level software:}{Prototyped two production software in AWS: a NGS annotatation pipeline (Snakemake) and a R\&D experiment management web portal}      
      \resumeItem{Experiments:}{Performed Nanopore long-read sequencing with the R\&D team to profile hydroponic metagenome}
      \resumeItem{Other wet lab experiments:}{Contributed to other R\&D experiments in crop phenotyping and sample collection}
    \resumeItemListEnd
    \resumeSubheading
    {Hong Kong Baptist University}{Research Assistant}
    {Hong Kong}{July 2015 - June 2018}
      \resumeItemListStart
      \resumeItem{Computational modeling:}{Built novel machine learning models based on biological neural networks and non-equilibrium statistical physics to conduct signal computation}
      \resumeItem{}{Implemented novel optimization algorithms in C\texttt{++} and Python to model \textit{C. elegans} neurons}
      \resumeItem{Data-driven research:}{Collaborated in three data-driven projects with interdisciplinary teams spanning four research labs}       
  \resumeItemListEnd
  \resumeSubHeadingListEnd

\vspace{-10pt}

%--------PROGRAMMING SKILLS------------
\section{Skills}
 \resumeSubHeadingListStart
 \resumeSubItem{Full-stack bioinformatics:}{Nextflow, NGS (amplicon, target enrichment,whole-genome sequencing, UMI); extraction, library prep, common wet lab; microcontrollers (Arduino, Raspberry Pi)}
  \resumeSubItem{Full-stack data science:}{machine learning, statistics, high-dimensional data analysis; data engineering, dashboards (NiceGUI/Dash), databases (Postgresql); AWS, Django, REST APIs, CI/CD, Git; Python, Rust, SQL, R, Bash, Java, Groovy, JavaScript, C/C++, \LaTeX; project management}
   \resumeSubItem{Computational biology:}{cancer biology, liquid biopsy, statistical physics}

   
 \resumeSubHeadingListEnd


  \section{Other Experience}
  \resumeSubHeadingListStart
    \vspace{-1pt}\item[]
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{The Abdus Salam International Centre for Theoretical Physics} &  {\textit{Mar 2018}} \\ Spring College on the Physics of Complex Systems (Trieste, Italy) & 
    \end{tabular*}\vspace{-5pt}
    \resumeItemListStart
    \resumeItem{Coursework:}{Completed graduate-level courses with the highest grade in reinforcement learning, statistical physics, protein biophysics, and computational neuroscience}
    \resumeItemListEnd
    \resumeSubheading
    {Iowa State University}{Exchange Student}
    {Ames, Iowa, United States}{August 2016 - December 2016}
      % \resumeItemListStart
      % \resumeItem{}{Completed coursework in physics, mathematics, and computer science.}
      % \resumeItemListEnd

    \resumeSubheading
    {Hong Kong Baptist University}{Teaching Assistant}
    {Hong Kong}{July 2016 - June 2018}
      % \resumeItemListStart
      % \resumeItem{}{Taught discussion sessions of Introduction to Physics for two semesters}
      % \resumeItemListEnd

  \resumeSubHeadingListEnd


 \section{Publications}

 \resumeSubHeadingListStart
     \item[] Xavier Bower, Jan Wignall, Joyce Zhu, Michael O’Sullivan, Naomi E. Searle, Lenny K. Hong, Matthew G. Varga, Tiffany E. Farmer, Emilio Rosas-Linhard, \textbf{Zeqian Li}, Jason Luong, Esther Lin, Marie Erica Simon, David S. Tsao, John R. ten Bosch, Gary Palmer MD, Ajeet Gajra MD, Chanh Huynh MD, Wen Zhou ``Validation of a liquid biopsy assay with increased sensitivity for clinical comprehensive genomic profiling.''  \textit{Manuscript submitted for publication.} (2024)
     \item[] Angela Hsiao, Brian Woodward, Patrick Ye, Matthew G Varga, Ghaith Altaie, Kevin Lu, Naomi Searle, Robb Viens, Sydne Langpap, \textbf{Zeqian Li}, Gary Palmer, Hatim Husain. ``Brief Report: Methylation-Based ctDNA Serial Monitoring Correlates with Therapeutic Response in Lung Cancer.''  \textit{Manuscript submitted for publication.} (2024)
     \item[] \textbf{Zeqian Li}, Ahmed Selim, Seppe Kuehn. ``Statistical prediction of microbial metabolic traits from genomes.''  \textit{PLOS Computational Biology} 19.12 (2023): e1011705.
     \item[] Kyle Crocker, Milena Chakraverti-Wuerthwein, \textbf{Zeqian Li}, Madhav Mani, Karna Gowda, Seppe Kuehn. ``Genomics patterns in the global soil microbiome emerge from microbial interactions.'' \textit{Nature Microbiology} (2024): 838-853.
     \item[] Chandana Gopalakrishnappa, \textbf{Zeqian Li}, Seppe Kuehn. ``Environmental modulators of algae-bacteria interactions at scale.'' \textit{Cell Systems} 15.9 (2024): 838-853.
     \item[] \textbf{Zeqian Li}, Vaibhhav Sinha, and Seppe Kuehn. "Constraints on microbial metabolic complexity." \textit{Nature Microbiology} 8.10 (2023): 1756-1757.
     \item[] Luis Miguel de Jesús Astacio$^*$, Kaumudi H. Prabhakara$^*$, \textbf{Zeqian Li}, Harry Mickalide, Seppe Kuehn. ``Closed microbial communities self-organize to persistently cycle carbon.''  \textit{Proceedings of the National Academy of Sciences} 118, no. 45 (2021): e2013564118. 
 \resumeSubHeadingListEnd

\section{Selected conference presentations}
\resumeSubHeadingListStart
     \item[] \textbf{Speaker:} Microbiome Research Symposium, The University of Chicago, ``Machine learning predicts microbial metabolic traits from genomes.''  (2023)
     \item[] \textbf{Speaker:} The American Physical Society March Meeting, ``Unique functional structure of the Yellowstone hot spring microbial mats revealed by multi-omics studies.''  (2022)
     \item[] \textbf{Speaker:} The Yellowstone Hot Spring Microbial Mats Symposium, Carnegie Institution for Science, ``Co-expression constrains genome organization in an extensively recombined microbial population.''  (2022)
     

     
 \resumeSubHeadingListEnd



%-------------------------------------------
\end{document}
